RecoParticleFlow/PFProducer/python/chargedHadronPFTrackIsolation_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+
+chargedHadronPFTrackIsolation = cms.EDProducer(
```

Comment:
this file is redundant and can be removed.
It is already provided by the framework with the call ``` descriptions.add("chargedHadronPFTrackIsolation", desc); ``` available in ChargedHadronPFTrackIsolationProducer